modules/proto/rollkitsdk/network/v1/types.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package rollkitsdk.network.v1;
+
+option go_package = "github.com/rollkit/go-execution-abci/modules/network/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/duration.proto";
+
+// Params defines the parameters for the network module.
+message Params {
+  // epoch_length is the number of blocks per attestation window
+  uint64 epoch_length = 1;
+  
+  // quorum_fraction is the voting power needed for soft-finality (e.g., "2/3")
+  string quorum_fraction = 2 [(cosmos_proto.scalar) = "cosmos.Dec"];
+  
+  // min_participation is the ejection threshold inside an epoch (e.g., "1/2")
+  string min_participation = 3 [(cosmos_proto.scalar) = "cosmos.Dec"];
+  
+  // prune_after is the number of epochs to retain attestation bitmaps
+  uint64 prune_after = 4;
+  
+  // emergency_mode is a governance switch to bypass quorum
+  bool emergency_mode = 5;
+  
+  // sign_mode determines when validators must sign
+  SignMode sign_mode = 6;
+}
+
+// SignMode defines when validators must sign
+enum SignMode {
```

Comment:
Do we need to support so many?

Reply:
👍 I will open an issue to add them later if needed